src/assets/stylesheets/Notifications.scss

```diff
@@ -33,7 +33,12 @@
     width: 100%;
     font-weight: $font-weight-bold;
     inset-block-start: 20%;
+
+    .Toastify__toast-icon {
```

Comment:
I wonder if you could play with the margin instead of the width to make it consistent with that same icon within top-bottom-center class https://github.com/RaspberryPiFoundation/editor-ui/pull/707/files#diff-542295bff42144fae5b97459b41cbb0acbec2b9b0b140864015b4e0fab1f0a55R24

Reply:
Thanks for this - I've taken a look, but couldn't get it working with the same margins as the bottom centre one (not sure why 😕 )